### DISCUSSION TITLE
Change Css on action

### DISCUSSION THREAD

**Discussion by Sahar-magh** [dowjones/react-dropdown-tree-select#562] [Q&A]
Is it possible to change css of the fa-icon after clicking on it ? I would love to mark a main selected item through an action on Fa icon. 
This doesn't work
`.action:visited {
  color: blue;
}
`thank you  :
![CaptureExemple](https://user-images.githubusercontent.com/40029558/166696697-c4560ec8-a716-4632-988f-8bd383579b26.PNG)
)

**Comment by mrchief** [comment]
visited might not work if it's not an `a` element. You might wanna try `:active`. Based on your use case, this may give you some pointers https://codepen.io/andreinacio/pen/doKGjP
